src/app/admin/services/event-mappings/event-mappings.service.spec.ts

```diff
@@ -77,4 +78,111 @@ describe('EventMappingsService', () => {
     expect(eventHandlers!.length).toBe(2);
     expect(eventHandlers!).toEqual(mockHandlers);
   });
+
+  it('should create event mapping', () => {
+    const mockEventMappingForm: EventMappingFormValues = {
+      type: '1000',
+      subType: '1001',
+      eventName: 'Prosecution opened',
+      eventHandler: 'StandardEventHandler',
+      withRestrictions: false,
+    };
+
+    const mockResponse: EventMappingData = {
+      id: 1,
+      type: '1000',
+      sub_type: '1001',
+      name: 'Prosecution opened',
+      handler: 'StandardEventHandler',
+      is_active: true,
+      has_restrictions: false,
+      created_at: '2024-05-05T11:00:00Z',
+    };
+
+    service.createEventMapping(mockEventMappingForm).subscribe((response) => {
+      expect(response).toEqual(mockResponse);
```

Comment:
For these assertions within the subscribe callback, you need to wrap the test in fakeAsync( .. ) and put tick() at the very end otherwise you get those pesky false positives.